database-plugins/pom.xml

```diff
@@ -20,12 +20,12 @@
   <parent>
     <artifactId>hydrator-plugins</artifactId>
     <groupId>co.cask.hydrator</groupId>
-    <version>1.6.0-SNAPSHOT</version>
+    <version>1.6.0</version>
   </parent>
 
   <properties>
     <!-- properties for script build step that creates the config files for the artifacts -->
```

Comment:
any idea why some pom files have this, but like cassandra-plugins/pom.xml doesn't?